api.js

```diff
@@ -148,6 +148,8 @@
         return this.user.login('performer', username, password, callback);
       },
       fetchOwn        : [GENERATE_GET, 'performer'],
+      matches         : [GENERATE_GET, 'user/matches'],
```

Comment:
use routes in /user and don't put them in the performer namespace (performer routes are deprecated)